src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -33,28 +33,22 @@
  * Handles communication with the keyserver.
  */
 public class Rscc {
+  public static final int PACKAGE_SIZE = 10000; // needed, since a static method access it.
```

Comment:
@jpduloch I tried something like this (honestly, I can't remember the exact thing) and it throw me more errors. If you can assure that your code example works, I agree to implement it like this :+1: